src/Symfony/Bundle/FrameworkBundle/Command/SecretsDecryptToLocalCommand.php

```diff
@@ -47,6 +48,10 @@ protected function configure(): void
 When the option <info>--force</info> is provided, secrets that already exist in the local vault are overriden.
 
     <info>%command.full_name% --force</info>
+
+When the option <info>--exit</info> is provided, the command will return a non-zero exit code if any errors are encountered.
```

Comment:
```suggestion
When the <info>--exit</info> option is provided, the command will return a non-zero exit code if any errors are encountered.
```

same for the option above, thanks